examples/audio_player_interaction/macos/Runner/DebugProfile.entitlements

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>com.apple.security.app-sandbox</key>
+	<true/>
+	<key>com.apple.security.cs.allow-jit</key>
+	<true/>
+	<key>com.apple.security.device.audio-input</key>
+	<true/>
+	<key>com.apple.security.network.server</key>
```

Comment:
should this also include

	<key>com.apple.security.network.client</key>
	<true/>

Reply:
To run the debug version of the audio_player_interaction on macOS the network.client. key is not needed.

Reply:
Even when speech recognition is not running locally on the device?

Reply:
Correct, when I tested locally that key is not needed when running on device.